builders/get-started/networks/moonbase.md

```diff
@@ -57,7 +57,93 @@ To get DEV tokens, enter the following message, replacing `<enter-address-here->
+|:------------:|:------------:|:-------------------------------------------------:|
+| [Moonbase ERC20 Minter](https://moonbase-minterc20.netlify.app/){target=_blank} | ERC-20 Faucet  |[https://github.com/PureStake/moonbase-mintableERC20](https://github.com/PureStake/moonbase-mintableERC20){target=_blank} |
+| [Moonbeam Uniswap](https://moonbeam-swap.netlify.app/#/swap){target=_blank}  |  Uniswap V2 Fork  | [https://github.com/PureStake/moonbeam-uniswap](https://github.com/PureStake/moonbeam-uniswap){target=_blank}  |
+| [MoonLink Dashboard](https://moonlink-dashboard.netlify.app/){target=_blank}  | Chainlink Demo  | [https://github.com/PureStake/moonlink-dashboard](https://github.com/PureStake/moonlink-dashboard){target=_blank}  |
+| [MoonLotto Lottery](https://moonbase-moonlotto.netlify.app/){target=_blank}  | TheGraph Demo  | [Interface](https://github.com/PureStake/moonlotto-interface){target=_blank}, [Subgraph](https://github.com/PureStake/moonlotto-subgraph){target=_blank}  |
+| [Moonbeam WalletConnect](https://moonbeam-walletconnect-demo.netlify.app/){target=_blank}  | WalletConnect Demo  | [https://github.com/PureStake/moonbeam-walletconnect-demo](https://github.com/PureStake/moonbeam-walletconnect-demo){target=_blank}  |
+| [Moonbase ChainBridge](https://moonbase-chainbridge.netlify.app/transfer){target=_blank}  | Chainbridge Demo  | [https://github.com/PureStake/chainbridge-ui](https://github.com/PureStake/chainbridge-ui){target=_blank}  |
+| [MoonGas](https://moonbeam-gasinfo.netlify.app/){target=_blank}  | Gas Price Tracker  | [https://github.com/albertov19/moonbeam-gas-station](https://github.com/albertov19/moonbeam-gas-station){target=_blank}  |
+
+
+!!! note
+    These DApps are intended for demonstration purposes only and may be incomplete or unsuitable for production deployments. 
+
+### Moonbase ERC20 Minter {: #moonbase-erc20-minter } 
+
+The Moonbase ERC20 Minter enables you to mint a variety of ERC-20 test tokens corresponding to the 8 planets of the Solar System, and Pluto. To mint tokens, first press **Connect MetaMask** in the upper right hand corner. Then scroll to the Mint Tokens section and the choose desired ERC20 contract. Press **Submit Tx** and confirm the transaction in MetaMask. Each mint will grant you 100 tokens, and you can mint tokens for each contract once per hour.
+
+![ERC20 Minter](/images/builders/get-started/networks/moonbase/moonbase-3.png)
+
+### Moonbeam Uniswap {: #moonbeam-uniswap } 
+
+[Moonbeam Uniswap](https://moonbeam-swap.netlify.app/#/swap){target=_blank} is a fork of [Uniswap-V2](https://uniswap.org/blog/uniswap-v2){target=_blank} deployed to Moonbase Alpha. Notably, Moonbeam Uniswap allows developers to easily make a swap to acquire [cross chain assets](/builders/xcm/xc20/) such as xcKarura or xcUnits for XCM testing purposes. To perform your first swap, take the following steps:
+
+1. Connect your MetaMask Wallet and ensure you're on the Moonbase Alpha network
```

Comment:
I think there are a few places where this line is copy pasta'd... wallet doesn't need to be capitalized here